module/behaviour/skills/HeadBehaviourSoccer/src/HeadBehaviourSoccer.cpp

```diff
@@ -479,7 +479,7 @@ namespace module::behaviour::skills {
         // Rotate target angles to World space
         Eigen::Vector3d lookVector = headToIMUSpace * lookVectorFromHead;
         // Compute inverse kinematics for head direction angles
-        std::vector<std::pair<ServoID, double>> goalAngles = calculateCameraLookJoints(lookVector);
+        std::vector<std::pair<ServoID, double>> goalAngles = calculateHeadJoints(lookVector);
 
         Eigen::Vector2d result;
         for (auto& angle : goalAngles) {
```

Comment:
While we are here, is this really the best way to extract the angles from std::vector<std::pair<ServoID, double>> into a Vector2d. Seems like a very strange way to go about it?

Reply:
I can't think of any better way, since we don't really know where the head servos are in the vector. Did you have an idea?

Reply:
Make an issue if it's worth making an issue